shared/telemetry/TelemetryUtils.js

```diff
@@ -135,27 +135,41 @@ export function getAppInfoEventData() {
   };
 }
 
-let passcodeRetryCount = 1;
+let retryCount = 1;
```

Comment:
we should set retryCount as 0 by default

Reply:
done